Tools/autotest/arduplane.py

```diff
@@ -530,7 +530,7 @@ def fly_mission(self, filename, mission_timeout=60.0, strict=True, quadplane=Fal
         num_wp = self.load_mission(filename, strict=strict)-1
         self.set_current_waypoint(0, check_afterwards=False)
         self.change_mode('AUTO')
-        self.wait_waypoint(1, num_wp, max_dist=60)
+        self.wait_waypoint(1, num_wp, max_dist=60, timeout=mission_timeout)
```

Comment:
This change is causing the autotest suite to fail.  I suggest removing it from this PR and raising it separately - unless it is vital for this PR somehow?!

Reply:
It is . . . because I need to have a lengthy loiter time for the baro offset to build up. The issue is that fly each frame does not specify a mission timeout, so the default 60s is used and passed into wait_waypoint, overriding _its_ default of 400s. 
For now I've specified to use 400s to restore the original behaviour.